src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-760.mdx

```diff
@@ -0,0 +1,23 @@
+---
+subject: Android agent
+releaseDate: '2024-09-10'
+version: 7.6.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.6.0.zip'
+---
+
+## New
+
+* Upgrade JDK support to JAVA22.
+* Upgrade AGP support to 8.6.
+* Add feature flag for event persistence. You can read our docs to learn how to [enable this feature](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-event-persistence.
+
+## Fix
+
+* Fix a corner case to prevent mismatch between appbuildid and appversion.
+* Fix where crashes are not ingested due to empty stacktraceElements.
+* Fix a NullValuePointer exception when shutdown is called.
+* Fix a parameter mismatch in gradle plugin which has caused confusion in the logs.
```

Comment:
```suggestion
* Fix a corner case to prevent mismatch between `appbuildid` and `appversion`.
* Fix where crashes are not ingested due to empty `stacktraceElements`.
* Fix a `NullValuePointer` exception when shutdown is called.
* Fix a parameter mismatch in gradle plugin which has caused confusion in the logs.
```